pkg/manifests/manifests.go

```diff
@@ -331,6 +331,11 @@ func (f *Factory) AlertmanagerMain(host string, trustedCABundleCM *v1.ConfigMap)
 
 	if trustedCABundleCM != nil {
 		volumeName := "alertmanager-trusted-ca-bundle"
+		for in, container := range a.Spec.Containers {
```

Comment:
nit: isn't the container index deterministic. we rely on this invariant in a couple of other places.

Reply:
I would not like to rely on an order of the containers, even if we are doing it other places. Don't see a problem with this current implementation. I would even suggestion to not make it deterministic, as its bad practice IMHO, as we clearly all don't know that. :D